app/views/badges/redeem.slim

```diff
@@ -19,18 +19,12 @@
           = "O código expirou!"
         - when NilClass
           #badge
-            #badge-name
-              |  Parabéns! Acabaste de ganhar a badge 
-              = @badge.name
-              | . 
             #badge-logo
-              |  <img src="
-              = @badge.logo_url
-              | " alt="
-              = @badge.name
-              | " class="img-responsive"> 
-            .col-md-12
-              = content_tag  :div, "", id: "share-button", data: { description:"Ganhei a badge de " + @badge.name + "!", image: "http://intra.seium.org" + @badge.logo_url}
+              = link_to edition_badge_path(@edition, @badge)
+                = image_tag @badge.logo_url, alt: @badge.name, class: "img-responsive"
+            p.text-center#badge-name
+              | Parabéns! Acabaste de ganhar a badge
```

Comment:
@frmendes I'm thinking about making this easily modifiable, since there are a lot of different opinions on the translation, should i create a global variable in the application controller with the translation or should i create a helper method that just returns the string of the translation? Or is there a better way that I don't know of?